operations/app/terraform/modules/data_factory/linked_services.tf

```diff
@@ -24,7 +24,7 @@ resource "azurerm_data_factory_linked_service_azure_blob_storage" "pdi_datasa" {
   data_factory_id          = azurerm_data_factory.pdi.id
   integration_runtime_name = azurerm_data_factory_integration_runtime_azure.pdi.name
   use_managed_identity     = true
-  service_endpoint         = "https://${var.resource_prefix}datasa.blob.core.windows.net"
+  service_endpoint         = "https://${var.resource_prefix}datasa${var.environment == "skylight" ? "1" : ""}.blob.core.windows.net"
```

Comment:
This is neat syntax. Was there already a `piskylightdatasa` storage account?

Reply:
We had one in the other "Skylight PHDI" tenant. I deleted it, but for some reason when I tried to recreate it in the new tenant, it said the name was already taken. 🤷